apps/desktop/src/components/right-panel/components/chat/chat-input.tsx

```
@@ -361,8 +385,42 @@ export function ChatInput(
+    const isHyprCloudEnabled = hyprCloudEnabledQuery.data;
+
+    if (isHyprCloudEnabled) {
+      return "HyprCloud";
+    }
+
+    switch (connectionType) {
+      case "Custom":
+        return customLlmModelQuery.data || "Custom Model";
+      case "HyprLocal":
+        return "Local LLM";
+        return "HyprLLM";
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)